impala/_rpc/thrift_util.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python
+# Copyright (c) 2012 Cloudera, Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Thrift utility functions
+
+from impala._thrift_api import (TSocket, TBufferedTransport)
+
+import getpass
+import sasl
```

Comment:
Get rid of this import.  It gets imported later on when needed.  And users may have not installed `python-sasl` on their systems.